src/types.ts

```diff
@@ -1423,7 +1423,7 @@ export type ReactionFilters<StreamChatGenerics extends ExtendableGenerics = Defa
 export type ChannelFilters<StreamChatGenerics extends ExtendableGenerics = DefaultGenerics> = QueryFilters<
   ContainsOperator<StreamChatGenerics['channelType']> & {
     members?:
-      | RequireOnlyOne<Pick<QueryFilter<string>, '$in' | '$nin'>>
+      | RequireOnlyOne<Pick<QueryFilter<string>, '$in'>>
```

Comment:
```suggestion
      | RequireOnlyOne<Pick<QueryFilter<string>, '$in' | '$nin'>>
```

Reply:
Removing `$nin` here will technically make this a breaking change

<img width="951" alt="Screenshot 2024-09-13 at 14 38 13" src="https://github.com/user-attachments/assets/0b48c70f-d7d9-4d16-b818-732783f223bc">

Reply:
In that case, I will keep it here for now

Reply:
Thanks